child_compassion/model/child_compassion.py

```diff
@@ -357,6 +359,32 @@ def _create_empty_case_study(self, cr, uid, child, context=None):
             return child_prop_obj.create(cr, uid, vals, context)
         return True
 
+    def update_delegate(self, cr, uid, context=None):
+        obj_children = self.pool.get('compassion.child')

```

Comment:
Since you are in the compassion.child object, you don't need to have obj_children because, self = obj_children.